llvm/utils/TableGen/X86RecognizableInstr.h

```diff
@@ -155,6 +155,8 @@ namespace X86Local {
   enum {
     AdSize16 = 1, AdSize32 = 2, AdSize64 = 3
   };
+
+  enum { ExplicitREX2 = 1 };
```

Comment:
No need enum for one entry.

Reply:
If no this enum, then in X86RecognizableInstr.cpp, the code would be
```
  ExplicitREX2Prefix =
      byteFromRec(Rec, "explicitOpPrefixBits") == 1;
```
which was hard to read.